presto-main/src/main/java/com/facebook/presto/sql/analyzer/Analysis.java

```diff
@@ -103,6 +105,17 @@ public void setQuery(Query query)
         this.query = query;
     }
 
+    public Optional<Explain> getExplainAnalyze()
+    {
+        return explainAnalyze;
+    }
+
+    public void setExplainAnalyze(Explain explainAnalyze)
```

Comment:
I was just following the way that `Insert` works. Do you think that `Insert` should also be converted to a `getStatement()` call?